page/index.html

```diff
@@ -1,99 +1,158 @@
+	<link href='https://fonts.googleapis.com/css?family=Lora:900,400,400italic' rel='stylesheet' type='text/css'>
 	<link rel="stylesheet" href="style.css">
 </head>
+
 <body>
 	<header>
 		<h1>CSS OFF</h1>
 	</header>
-	<article>
+	<article class="box-model">
 		<figure>
 			<img src="../images/barbie.png" alt="">
 		</figure>
 		<h1>Box Model</h1>
 		<section>The Barbie doll was invented in 1959 by Ruth Handler (co-founder of Mattel), whose own daughter was called Barbara. Barbie was introduced to the world at the American Toy Fair in New York City. Barbie's job was teenage fashion doll. The full name fo the first doll was Barbie Millicent Roberts, from Willows, Wisconsin.</section>
 	</article>
-	<article>
-		<figure>
-			<img src="../images/Photo.png" alt="">
-		</figure>
+	<article class="positioning-relative">
+		<div class="volume-number">VOLUME 3 - SEPTEMBER 2013</div>
+		<a href="http://www.barbie.com/en-us">
+			<figure>
+				<img src="../images/Photo.png" alt="">
+			</figure>
+		</a>
 		<p>FEATURE</p>
 		<h1>POSITIONING RELATIVES</h1>
 		<section>In 1958, English pictorial photographer Henry Peach Robinson created the world's first photomontage by combining five different negatives to make one complete print of a young girl on her deatbed. "Fading Away" -- Robinson's first and most famous composite photo -- depicts a young girl dying of consumption and was controversial when it was exhibited, with many believing it was not a suitable subject for photography.</section>
 	</article>
-
-	<article>
+	<article class="staying-afloat">
 		<h1>Staying afloat</h1>
 		<section>Lobsterman John Aldridge fell into the shark-infested water of Long Island while his crewmates were asleep. He was kept afloat for more than 12 hours by his rubber boots, which he used as flotation devices. The coast guard rescued him 43 miles south of Montauk where he fell off his vessel.</section>
-		<figure>
-			<img src="../images/boots.png" alt="">
-		</figure>
+		<a href="http://www.bootcity.com/">
+			<figure>
+				<img src="../images/boots.png" alt="">
+			</figure>
+		</a>
 	</article>
-	<article>
-		<h1>LIST STYLE: BULLETS.</h1>
-		<h3>A ROUND-UP OF POPULAR AMMUNITION</h3>
+	<article class="list-style">
+		<div class="bullet_header">
+			<h1>LIST STYLE: BULLETS.</h1>
+			<h3>A ROUND-UP OF POPULAR AMMUNITION</h3>
+		</div>
 		<figure>
 			<img src="../images/bullets.png" alt="">
 		</figure>
 		<ul>
-			<li><div class="bullet-larg-text">.223</div> REMINGTON 69gr BTHP</li>
-			<li><div class="bullet-larg-text">.308</div> WINCHESTER 168gr BTHP</li>
-			<li>7.62x39mm RUSSIAN 154gr JSP</li>
-			<li><div class="bullet-larg-text">7.62</div> THUMPER 220gr RN</li>
-			<li><div class="bullet-larg-text">7.62</div> THUMPER 240gr BTHP</li>
+			<li class="bullet-one">
+				<div class="bullet-med-text">.223</div><span class="brand-small">REMINGTON</br></span><span class="bullet-small-text">69gr BTHP<span></li>
```

Comment:
It just feels like you are using a lot of extra DOM just to create presentation. Good opportunity to refactor this.